### ISSUE TITLE
ISO19139 codelists aren't displayed in template

### ISSUE THREAD

**Issue by lushc** [GeoViQua/geoviqua-geonetwork-plugin#6]
Codelist drop-down menus for GVQ elements are populated with codelist values, e.g.

``` xml
<gvq:purpose>
    <gvq:GVQ_PublicationPurposeCode codeList="http://schemas.geoviqua.org/GVQ/4.0/resources/Codelist/gvqCodelists.xml#GVQ_PublicationPurposeCode" codeListValue=""/>
</gvq:purpose>
```

but ISO19139 codelist values aren't shown:

``` xml
<gmd:dateType>
    <gmd:CI_DateTypeCode codeList="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/Codelist/ML_gmxCodelists.xml#CI_DateTypeCode" codeListValue=""/>
</gmd:dateType>
```

**Comment by lushc** [comment]
http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/Codelist/ML_gmxCodelists.xml results in a 404. Might have to change the URL to http://www.isotc211.org/2005/resources/Codelist/ML_gmxCodelists.xml? Although I don't think the issue is that the link is 404, because iso19139 documents still display the codelists.

I think the root cause of the problem is the plugin loading multiple codelists and merging them into one variable. I remember before if there was a conflict then the whole codelist wouldn't be loaded.

**Comment by lushc** [comment]
Fixed by b9981749e19d4c66b394ca2e1e9575a7e9f1eeda.
